ng.go

```diff
@@ -83,6 +84,7 @@ func main() {
 	shell.Init()
 
 	flagJupyter := flag.String("jupyter", "", "path to jupyter kernel connection file")
+	flag.BoolVar(&startInShell, "shell", false, "start in shell mode")
```

Comment:
instead of yet another global variable, perhaps declare as a local variable, _e.g._ `flagShell`, that would then be passed as an argument to `loop(v)` ?